security-council-rehearsals/templates/r5-guardian-override/.env

```diff
@@ -0,0 +1,9 @@
+# Modify these values according to the Rehearsal instructions
+COUNCIL_SAFE=
+GuardianSafe_ADDRESS=
+SuperchainConfig_ADDRESS=
+
+# Do not edit
+SAFE_NONCE=""
+DUMMY_DEPUTY_GUARDIAN_MODULE=0xAbCD1234ABCD1234AbCD1234ABcD1234ABCD1234
```

Comment:
Why is this address hardcoded, but not the above addresses? A DGM only can be used with 1 safe, but since we aren't actually using the DGM in the rehearsal—just rmeoving it—I'm guessing we reuse this DGM regardless of the above addresses?

Reply:
I see, this is not even a real DGM, it's just a made up address